### DISCUSSION TITLE
Welcome to MMM-CalendarExt3 Discussions!

### DISCUSSION THREAD

**Discussion by eouia** [MMRIZE/MMM-CalendarExt3#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PaekHyun** [comment]
font size를 크게 하려면 어떻게 해야 할까요?

main.css에
:root {

 --font-size : 20px;
 --font-size-xsmall : 1rem;
 --font-size-small : 1.25rem;
 --font-size-medium :1.75rem;
 --font-size-large : 3.5rem;
 --font-size-xlarge : 4rem;
}

라고 써 넣었는데 다른모듈은 적용이 되는데
calendarext3와 ex3timeline은 적용이 안되네요.

마찬가지로 같은 내용을 
custum.css에도 넣었는데 마찬가지로 안되는데 어디를 어떻게 수정해야 할까요?
